sensorsafrica/management/commands/upload_to_ckan.py

```diff
@@ -104,18 +106,16 @@ def handle(self, *args, **options):
                     resource_name = "{month} {year} Sensor Data Archive".format(
                         month=calendar.month_name[date.month], year=date.year
                     )
-
-                    filepath = "/tmp/%s.csv" % resource_name.lower().replace(" ", "_")
-
-                    self._write_file(filepath=filepath, qs=qs)
+                    temp = self._write_file(qs=qs)
+                    filepath = temp.name
```

Comment:
Lets create the file here. Code is always easier to follow if the resource creator is also the one that disposes it.
```suggestion
                    fp = tempfile.NamedTemporaryFile(mode="w+b", suffix=".csv")
                    self._write_file(fp, qs)
                    filepath = fp.name
```